server/BUILD

```diff
@@ -132,23 +130,23 @@ java_deps(
     target = ":server-bin-mac",
     java_deps_root = "server/lib/",
     visibility = ["//:__pkg__"],
-    maven_name = True,
+    maven_name = False,
```

Comment:
We don't package the JARs using 'maven' naming conventions anymore: this allows us to produce an archive that contains the dependencies pulled in by `grpc` build rules that don't conform to maven tagging conventions.